KsApi/GraphAPI.swift

```diff
@@ -0,0 +1,1039 @@
+// @generated
```

Comment:
Was this entirely auto generated? Did you manually edit a lot of this to tailor to our project? Trying to get a feel for what Apollo is doing for us.

Reply:
Auto-generated by Apollo during compile time, not a user-editable file.